src/codeu/chat/common/ServerInfoTest.java

```diff
@@ -0,0 +1,22 @@
+package codeu.chat.common;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public class ServerInfoTest {
+
+    @Test
+    public void testServerInfo() {
+        ServerInfo server1 = new ServerInfo();
```

Comment:
The way I understood this, was the 1.0.0 was the default version if there's no parameters. If the server version is changed, which is when there are now parameters, the version will be what the parameters are. Should I leave the empty parameter test case as is, but then just use a variable for the version and check accordingly so that the version can be changed for the test case with the parameter?